templates/CRM/Activity/Form/ActivityView.tpl

```diff
@@ -9,7 +9,7 @@
 *}
 <div class="crm-block crm-content-block crm-activity-view-block">
       {if $activityTypeDescription}
-        <div class="help">{$activityTypeDescription}</div>
+        <div class="help">{$activityTypeDescription|escape}</div>
```

Comment:
no direct reason  - but the temptation to change how this is loaded is there so this is hardening in case that happens

Reply:
Going to try to look at this this weekend but just noting that this field is html so purify seems more better.

Reply:
fixed